src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery {
+    private ColorSupplier colorSupplier;
+
+    public Lottery(ColorSupplier colorSupplier) {
+        this.colorSupplier = colorSupplier;
+    }
+
+    public Ball getRandomBall() {
+        Color randomColor = colorSupplier.getRandomColor();
+        int randomNumber = new Random().nextInt(100) + 1;
```

Comment:
your code recreates new Random instance per each method invocation - let's move random to the class scope instead to avoid this 
Also, 100 should be a class constant. Don't forget about constant case (in naming), access modifiers etc